docs/azure/sdk/resource-management.md

```diff
@@ -74,11 +74,37 @@ using System.Threading.Tasks;
 
 // Code omitted for brevity
 
-var armClient = new ArmClient(new DefaultAzureCredential());
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
 ```
 
 For more information about the `Azure.Identity.DefaultAzureCredential` class, see [DefaultAzureCredential Class](/dotnet/api/azure.identity.defaultazurecredential).
 
+### Management SDK Cheat Sheet
+To quickly get start with our new version Azure management SDK for .NET in less than 5 minutes, imagin you have a task to create/list/update/delete your service bus namespace(a typical kind of Azure resource), below is all the code needed. Just two steps needed.
+First, authenticate and select the subscription & resource group that you want to work on.
+```csharp
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.ServiceBus;
+
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+SubscriptionResource subscription = armClient.GetDefaultSubscription();
+ResourceGroupResource resourceGroupResource = client.GetDefaultSubscription().GetResourceGroup(resourceGroupName).Value;
```

Comment:
Since we have the implicit operator between `Response<T>` and `T`, actually we could omit the `.Value` here if it does not cause confusion